sam_handler/handler.py

```diff
@@ -191,7 +191,11 @@ def handle(self) -> str:
         self._create_generation_destination()
         template_path = self._prepare_template()
         self._build(invoker, template_path)
-        if template_path != self.sam_template_path:
+
+        build_args = self.arguments.get('build_args', {})
+        delete_temporary_jinja_file = build_args.get('delete_temporary_jinja_file', True)
+
+        if delete_temporary_jinja_file is True and template_path != self.sam_template_path:
```

Comment:
It's already a boolean
```suggestion
        if delete_temporary_jinja_file and template_path != self.sam_template_path:
```

Reply:
Ack